src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala

```diff
@@ -115,6 +118,23 @@ class WorkflowManagerActor(dataAccess: DataAccess, backend: Backend) extends Act
     }
   }
 
+  private def assertCallFqnWellFormed(callFqn: FullyQualifiedName): Future[(String, String)] = {
+    callFqn.split("\\.").toSeq match {
+      case s:Seq[String] if s.size >= 2 => Future.successful(s.head, s.last)
+      case _ => throw new UnsupportedOperationException("Expected a fully qualified name to have two parts")
+    }
+  }
+
+  private def callStdoutStderr(workflowId: WorkflowId, callFqn: String): Future[(WdlFile, WdlFile)] = {
+    for {
+      _ <- assertWorkflowExistence(workflowId)
+      _ <- assertCallExistence(workflowId, callFqn)
+      (wf, call) <- assertCallFqnWellFormed(callFqn)
+      stdout <- Future.successful(backend.stdout(workflowId, wf, call))
+      stderr <- Future.successful(backend.stderr(workflowId, wf, call))
+    } yield (stdout, stderr)
+  }
```

Comment:
Yeah I don't think we're getting that now, I think this needs to be checked with `getExecutionStatus`.